deploy/basic/terraform/outputs.tf

```diff
@@ -19,11 +19,17 @@ output "generated_private_key_pem" {
 }
 
 output "dev" {
-  value = "Made with \u2764 by Oracle A-Team"
+  value = "Made with \u2764 by Oracle A-Team Alumni"
```

Comment:
😆 .  But...

Reply:
accepting suggestions that are more close to reality.

Reply:
After voting, we will keep like this for now. Can re-visit later.